modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -236,12 +236,8 @@ def form_is210966
       end
 
       def form_is264555_and_should_use_lgy_api
-        # TODO: Remove prod/test check and ALWAYS require icn
-        if Rails.env.production? || Rails.env.test?
-          params[:form_number] == '26-4555' && icn
-        else
-          params[:form_number] == '26-4555'
-        end
+        # TODO: Remove comment octothorpe and ALWAYS require icn
+        params[:form_number] == '26-4555' # && icn
```

Comment:
Do we need to keep the commented out ` && icn` here?

Reply:
I want to keep it in as a reminder that we ultimately want the `&& icn` in there. This is just a temporary situation to make testing on staging easier.

Reply:
Aaaah, I didn't know what `octothorpe` meant but google has taught me. Thanks!